src/VCEL.CSharp/Expression/Func/DefaultCSharpFunctions.cs

```diff
@@ -54,7 +83,18 @@ public void Register(string name, Func<object[], string> func)
         private void Register(string name, Func<object[], IContext<string>, string> func, params IDependency[] deps)
             => functions[name.ToLower()] = new Function<string>(func, deps);
 
+        public void RegisterEnsureOneArg(string name, Func<object, string> func)
+            => Register(name, (args, _) => args?.Length == 1 && args[0] == null ? null : func(args[0]), new FuncDependency(name));
```

Comment:
Should be `args?.Length != 1 || args[0] == null` ?

Reply:
Though this check will fail if args.Length == 0
`args?.Length == 1 && args[0] != null ? func(args[0]) : null`